model/src/main/java/de/danoeh/antennapod/model/feed/FeedItem.java

```diff
@@ -42,6 +45,7 @@ public class FeedItem extends FeedComponent implements Serializable {
     private transient Feed feed;
     private long feedId;
     private String podcastIndexChapterUrl;
+    private Hashtable<String, String> podcastIndexTranscriptUrls;
```

Comment:
If we only store one of them in the database, how about just storing one here as well? With a method like `updateTranscriptIfBetterFormat(String type, String url)`, you could always update if the new one is JSON, but only update with srt if none is set. Otherwise future developers might get confused about the fact that there is only one url in the hash map.

Reply:
good suggestion, please review again